src/Command/ContaoFrameworkDependentInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+namespace Contao\CoreBundle\Command;
+
+/**
+ * Interface FrameworkDependentCommandInterface
+ * Only boot Contao-Framework if Command implements this Interface
+ *
+ * @author Dominik Tomasi <https://github.com/dtomasi>
+ */
+interface ContaoFrameworkDependentInterface {
```

Comment:
I did also mean line breaks in the control structures (see https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#11-example for an example)